lib/xboxrt/libc_extensions/stdlib_ext_.h

```diff
@@ -17,6 +17,8 @@ typedef void (__cdecl *_purecall_handler)(void);
 _purecall_handler __cdecl _get_purecall_handler (void);
 _purecall_handler __cdecl _set_purecall_handler (_purecall_handler function);
 
+#define _sys_nerr _PDCLIB_ERRNO_MAX
```

Comment:
As usual, I think we should add a comment about *what* kind of extension this is.
For what API / standard and maybe a link to documentation.
\- I'm surprised I didn't mention this in my initial review.

Reply:
Done.